Assets/AppCenter/Editor/AppCenterSettingsMakerAndroid.cs

```diff
@@ -17,6 +17,9 @@ public class AppCenterSettingsMakerAndroid : IAppCenterSettingsMaker
     private const string UseCrashesKey = "appcenter_use_crashes";
     private const string UsePushKey = "appcenter_use_push";
     private const string UseAnalyticsKey = "appcenter_use_analytics";
+    private const string UseAuthKey = "appcenter_use_auth";
+    private const string CustomConfigUrlKey = "appcenter_custom_config_url";
+    private const string UseCustomConfigUrlKey = "appcenter_use_custom_config_url";
```

Comment:
It should be `appcenter_auth_*`

Reply:
Sure I'll add the auth prefix to all uses of the config url